client/src/client.rs

```diff
@@ -687,7 +711,20 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 
 				operation.op.update_cache(new_cache);
 
-				let (main_sc, child_sc, tx, _, changes_trie_tx) = storage_changes.into_inner();
+				let (main_sc,
+					child_sc,
+					offchain_sc,
+					tx, _,
+					changes_trie_tx,
+				) = storage_changes.into_inner();
+
+				if let Some(mut offchain_storage) = self.backend.offchain_storage() {
+					offchain_sc.iter().for_each(|(k,v)| {
+						offchain_storage.set(b"block_import_info", k,v)
```

Comment:
```suggestion
						offchain_storage.set(b"block_import_info",k, v)
```

Accessing this prefix will require changes to OCW storage APIs. Currently it's not supported to read arbitrary-prefix data.

Reply:
This also does not seem fully correct. We don't want to write directly to DB here, we should rather put this stuff to `opreation.op` or transaction, no so that it can be atomically flushed to the DB with all other changes.

Reply:
There is not really a need to have arbitrary data, I assume we can use the currently unused `LOCAL` variant of `StorageKind` (unless that implies a different semantic).

Reply:
It is meant to have different semantics. The block import should ideally write to fork-aware DB (`LOCAL`) ,but for now we need to make it write to `PERSISTENT` one as that's the only one implemented.